linux/user-and-file-management/bash-environment/linux-shell-variables.md

```diff
@@ -0,0 +1,86 @@
+---
+author: kapnobatai136
+
+aspects:
+  - introduction
+
+type: normal
+
+category: must-know
+
+tags:
+  - introduction
+  - linux
+  - variables
+
+---
+
+# Variables
+
+---
+## Content
+
+Think of *variables* as a box. 
+
+They are used to store information. To make them easily identifiable, you put a label on them. 
+
+Let's try creating a variable now:
+
+```bash
+best_app="Enki"
+```
+
+> **Note:** don't add any spaces around the `=` sign or you'll get an error saying `"Command not found"`.
+
+We've taken the `"Enki"` information, and stored it in our `best_app` box:
+
+![variable-mental-model](https://img.enkipro.com/fa537341f3027f1cea7b76ecc3398e9d.png)
+
+To print this variable, you'd type:
+
+```bash
+echo $best_app
+Enki
```

Comment:
For situations like this, should we maybe display the output as commented out? 

Right now it's weird because the output might actually also get syntax highlighted and become distracting.

How about we use this approach everywhere:

```suggestion
echo $best_app
# Enki
```

Otherwise, we can be explicit about the output and mention it after a sentence "this is the output"

Reply:
yes, I was also leaning towards commenting the output. I'll change them